src/index.ts

```diff
@@ -221,3 +245,7 @@ function evolveC<Struct extends object>(
         return newStruct as Struct;
     };
 }
+
+export const ctrlEnter = (e: KeyboardEvent) => e.ctrlKey && e.key === 'Enter';
+export const metaEnter = (e: KeyboardEvent) => e.metaKey && e.key === 'Enter';
+export const defaultPredicate = (e: KeyboardEvent) => ctrlEnter(e) || metaEnter(e);
```

Comment:
Should we really export this one?

Additionally, `ctrlEnter` and `metaEnter` is also exported, is it intended?
(If so, we may provide them through more sophisticated API.)

Reply:
OK, I decided not to export these values.

I add some JSDoc comments to the `form` function to explain how to use `customSubmission` option.

Reply:
Stopped exporting: https://github.com/herp-inc/cyclic-form/pull/2/commits/594d4f98bfc68f2bb40c3458f89ade5d93c459d1
Added JSDoc comment: https://github.com/herp-inc/cyclic-form/pull/2/commits/350fae9a1e6e3885e626aa6df2529060a10dcc5d